PerformanceTests/Media/SeekLatencyWhilePlaying.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html>
+<head>
+<script src="../resources/runner.js"></script>
+<script>
+var video;
+var url;
+
+window.addEventListener('load', () => {
+    PerfTestRunner.prepareToMeasureValuesAsync({
+        unit: 'ms',
+        done: function () {
+            if (video) {
+                video.src = null;
+                video.load();
+            }
+        }
+    });
+
+    runTest();
+});
+
+function waitFor(object, event) {
+    return new Promise(resolve => {
+        object.addEventListener(event, resolve, {once: true});
+    })
+}
+
+function sleepFor(timeout) {
+    return new Promise(resolve => setTimeout(resolve, timeout));
+}
```

Comment:
Nit: It would be useful to have these in a separate .js file so they can be shared